official/recommendation/data_pipeline.py

```diff
@@ -297,7 +297,12 @@ def make_input_fn(self, batch_size):
     """Create an input_fn which checks for batch size consistency."""
 
     def input_fn(params):
-      param_batch_size = (params["batch_size"] if self._is_training else
+      """Returns batches for training."""
+
+      # Estimator passes batch_size during training and eval_batch_size during
+      # eval. TPUEstimator only passes batch_size.
+      param_batch_size = (params["batch_size"] if self._is_training or not
```

Comment:
I think this would be a bit clearer as:
```
param_batch_size = (params["batch_size"] if self._is_training else
                    params.get("eval_batch_size") or params["batch_size"])
```

And the same below.